website/client/src/components/auth/login.vue

```diff
@@ -0,0 +1,244 @@
+<template>
+  <form
+    v-if="forgotPassword"
+    id="forgot-form"
+    @submit.prevent="forgotPasswordLink"
```

Comment:
are both `@submit` and `@keyup.enter` necessary? I think `@keyup.enter` might be redundant